### PR TITLE
working dockerfile + config 

### PR DESCRIPTION
**Pull request by rgraff**
These changes get the dockerfile working in development.

### CODE REVIEW

File: conf/database.sh [hunk 1]
```diff
@@ -10,31 +10,16 @@
...
-if [ "$INSP_DATABASE_PASSWORD" = "" ]; then
-    INSP_DATABASE_PASSWORD="stellar"
-fi
-
-if [ "$INSP_DATABASE_NAME" = "" ]; then
-    INSP_DATABASE_NAME="stellar_development"
-fi
-
 cat <<EOF
+<module name="sha1">
```
Comment (rgraff) [inline conf/database.sh:18]: I think this is what was preventing it working previously

File: Dockerfile [hunk 2]
```diff
@@ -4,16 +4,17 @@ LABEL maintainer1="Adam <adam@anope.org>" \
       maintainer2="Sheogorath <sheogorath@shivering-isles.com>"
 
 ARG VERSION=insp3
-ARG CONFIGUREARGS="--enable-extras=m_pgsql.cpp"
+ARG CONFIGUREARGS=
```
Comment (rgraff) [inline Dockerfile:7]: leave the args empty. Easier to test changes with docker-compose

**Comment by rgraff** [comment]
@silhouette88 @therubymug I'm going to merge this to unblock me but please look at the docker file changes that I made.

**Comment by silhouette88** [comment]
Thanks @rgraff, looks good to me. I was working on the load balancer configurations over the weekend and saw this was merged so built and deployed the latest image to Nomad staging. I’m still trying to get the websocket stuff working through the load balancers but I think I’m close.

**Pull request merged**